{"nodes":[{"content":"Thrown when an attempt is made to add an object to the identity cache by using a key that is already being used.","nodes":[{"pos":[0,112],"content":"Thrown when an attempt is made to add an object to the identity cache by using a key that is already being used.","nodes":[{"content":"Thrown when an attempt is made to add an object to the identity cache by using a key that is already being used.","pos":[0,112]}]}],"pos":[692,805],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class.","nodes":[{"pos":[0,100],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.Linq.DuplicateKeyException\"&gt;&lt;/xref&gt;</ph> class.","source":"Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class."}],"pos":[2836,2937],"yaml":true},{"content":"The duplicate key that caused the exception to be thrown.","nodes":[{"pos":[0,57],"content":"The duplicate key that caused the exception to be thrown.","nodes":[{"content":"The duplicate key that caused the exception to be thrown.","pos":[0,57]}]}],"pos":[3090,3148],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class by referencing the duplicate key and providing an error message.","nodes":[{"pos":[0,164],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.Linq.DuplicateKeyException\"&gt;&lt;/xref&gt;</ph> class by referencing the duplicate key and providing an error message.","source":"Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class by referencing the duplicate key and providing an error message."}],"pos":[4373,4538],"yaml":true},{"content":"The duplicate key that caused the exception to be thrown.","nodes":[{"pos":[0,57],"content":"The duplicate key that caused the exception to be thrown.","nodes":[{"content":"The duplicate key that caused the exception to be thrown.","pos":[0,57]}]}],"pos":[4707,4765],"yaml":true},{"content":"The message to appear when the exception is thrown.","nodes":[{"pos":[0,51],"content":"The message to appear when the exception is thrown.","nodes":[{"content":"The message to appear when the exception is thrown.","pos":[0,51]}]}],"pos":[4828,4880],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class by referencing the duplicate key, providing an error message, and specifying the exception that caused this exception to be thrown.","nodes":[{"pos":[0,231],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.Linq.DuplicateKeyException\"&gt;&lt;/xref&gt;</ph> class by referencing the duplicate key, providing an error message, and specifying the exception that caused this exception to be thrown.","source":"Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class by referencing the duplicate key, providing an error message, and specifying the exception that caused this exception to be thrown."}],"pos":[6189,6421],"yaml":true},{"content":"The duplicate key that caused the exception to be thrown.","nodes":[{"pos":[0,57],"content":"The duplicate key that caused the exception to be thrown.","nodes":[{"content":"The duplicate key that caused the exception to be thrown.","pos":[0,57]}]}],"pos":[6616,6674],"yaml":true},{"content":"The message to appear when the exception is thrown.","nodes":[{"pos":[0,51],"content":"The message to appear when the exception is thrown.","nodes":[{"content":"The message to appear when the exception is thrown.","pos":[0,51]}]}],"pos":[6737,6789],"yaml":true},{"content":"The previous exception that caused the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> exception to be thrown.","nodes":[{"pos":[0,122],"content":"The previous exception that caused the <ph id=\"ph1\">&lt;xref href=\"System.Data.Linq.DuplicateKeyException\"&gt;&lt;/xref&gt;</ph> exception to be thrown.","source":"The previous exception that caused the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> exception to be thrown."}],"pos":[6862,6985],"yaml":true},{"content":"Gets the object that caused the exception.","nodes":[{"pos":[0,42],"content":"Gets the object that caused the exception.","nodes":[{"content":"Gets the object that caused the exception.","pos":[0,42]}]}],"pos":[8028,8071],"yaml":true},{"content":"The object that caused the exception.","nodes":[{"pos":[0,37],"content":"The object that caused the exception.","nodes":[{"content":"The object that caused the exception.","pos":[0,37]}]}],"pos":[8181,8219],"yaml":true}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Data.Linq.DuplicateKeyException\n  commentId: T:System.Data.Linq.DuplicateKeyException\n  id: DuplicateKeyException\n  children:\n  - System.Data.Linq.DuplicateKeyException.#ctor(System.Object)\n  - System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)\n  - System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)\n  - System.Data.Linq.DuplicateKeyException.Object\n  langs:\n  - csharp\n  name: DuplicateKeyException\n  nameWithType: DuplicateKeyException\n  fullName: System.Data.Linq.DuplicateKeyException\n  type: Class\n  assemblies:\n  - System.Data.Linq\n  namespace: System.Data.Linq\n  summary: Thrown when an attempt is made to add an object to the identity cache by using a key that is already being used.\n  syntax:\n    content: 'public class DuplicateKeyException : InvalidOperationException'\n  inheritance:\n  - System.Object\n  - System.Exception\n  - System.InvalidOperationException\n  implements: []\n  inheritedMembers:\n  - System.Exception.Data\n  - System.Exception.GetBaseException\n  - System.Exception.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Exception.GetType\n  - System.Exception.HelpLink\n  - System.Exception.HResult\n  - System.Exception.InnerException\n  - System.Exception.Message\n  - System.Exception.SerializeObjectState\n  - System.Exception.Source\n  - System.Exception.StackTrace\n  - System.Exception.TargetSite\n  - System.Exception.ToString\n  - System.Object.Equals(System.Object)\n  - System.Object.Equals(System.Object,System.Object)\n  - System.Object.GetHashCode\n  - System.Object.MemberwiseClone\n  - System.Object.ReferenceEquals(System.Object,System.Object)\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object)\n  commentId: M:System.Data.Linq.DuplicateKeyException.#ctor(System.Object)\n  id: '#ctor(System.Object)'\n  parent: System.Data.Linq.DuplicateKeyException\n  langs:\n  - csharp\n  name: DuplicateKeyException(Object)\n  nameWithType: DuplicateKeyException.DuplicateKeyException(Object)\n  fullName: DuplicateKeyException.DuplicateKeyException(Object)\n  type: Constructor\n  assemblies:\n  - System.Data.Linq\n  namespace: System.Data.Linq\n  summary: Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class.\n  syntax:\n    content: public DuplicateKeyException (object duplicate);\n    parameters:\n    - id: duplicate\n      type: System.Object\n      description: The duplicate key that caused the exception to be thrown.\n  overload: System.Data.Linq.DuplicateKeyException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)\n  commentId: M:System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)\n  id: '#ctor(System.Object,System.String)'\n  parent: System.Data.Linq.DuplicateKeyException\n  langs:\n  - csharp\n  name: DuplicateKeyException(Object, String)\n  nameWithType: DuplicateKeyException.DuplicateKeyException(Object, String)\n  fullName: DuplicateKeyException.DuplicateKeyException(Object, String)\n  type: Constructor\n  assemblies:\n  - System.Data.Linq\n  namespace: System.Data.Linq\n  summary: Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class by referencing the duplicate key and providing an error message.\n  syntax:\n    content: public DuplicateKeyException (object duplicate, string message);\n    parameters:\n    - id: duplicate\n      type: System.Object\n      description: The duplicate key that caused the exception to be thrown.\n    - id: message\n      type: System.String\n      description: The message to appear when the exception is thrown.\n  overload: System.Data.Linq.DuplicateKeyException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)\n  commentId: M:System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)\n  id: '#ctor(System.Object,System.String,System.Exception)'\n  parent: System.Data.Linq.DuplicateKeyException\n  langs:\n  - csharp\n  name: DuplicateKeyException(Object, String, Exception)\n  nameWithType: DuplicateKeyException.DuplicateKeyException(Object, String, Exception)\n  fullName: DuplicateKeyException.DuplicateKeyException(Object, String, Exception)\n  type: Constructor\n  assemblies:\n  - System.Data.Linq\n  namespace: System.Data.Linq\n  summary: Initializes a new instance of the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> class by referencing the duplicate key, providing an error message, and specifying the exception that caused this exception to be thrown.\n  syntax:\n    content: public DuplicateKeyException (object duplicate, string message, Exception innerException);\n    parameters:\n    - id: duplicate\n      type: System.Object\n      description: The duplicate key that caused the exception to be thrown.\n    - id: message\n      type: System.String\n      description: The message to appear when the exception is thrown.\n    - id: innerException\n      type: System.Exception\n      description: The previous exception that caused the <xref href=\"System.Data.Linq.DuplicateKeyException\"></xref> exception to be thrown.\n  overload: System.Data.Linq.DuplicateKeyException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.Linq.DuplicateKeyException.Object\n  commentId: P:System.Data.Linq.DuplicateKeyException.Object\n  id: Object\n  parent: System.Data.Linq.DuplicateKeyException\n  langs:\n  - csharp\n  name: Object\n  nameWithType: DuplicateKeyException.Object\n  fullName: DuplicateKeyException.Object\n  type: Property\n  assemblies:\n  - System.Data.Linq\n  namespace: System.Data.Linq\n  summary: Gets the object that caused the exception.\n  syntax:\n    content: public object Object { get; }\n    return:\n      type: System.Object\n      description: The object that caused the exception.\n  overload: System.Data.Linq.DuplicateKeyException.Object*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\nreferences:\n- uid: System.InvalidOperationException\n  parent: System\n  isExternal: false\n  name: InvalidOperationException\n  nameWithType: InvalidOperationException\n  fullName: System.InvalidOperationException\n- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object)\n  parent: System.Data.Linq.DuplicateKeyException\n  isExternal: false\n  name: DuplicateKeyException(Object)\n  nameWithType: DuplicateKeyException.DuplicateKeyException(Object)\n  fullName: DuplicateKeyException.DuplicateKeyException(Object)\n- uid: System.Object\n  parent: System\n  isExternal: false\n  name: Object\n  nameWithType: Object\n  fullName: System.Object\n- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)\n  parent: System.Data.Linq.DuplicateKeyException\n  isExternal: false\n  name: DuplicateKeyException(Object, String)\n  nameWithType: DuplicateKeyException.DuplicateKeyException(Object, String)\n  fullName: DuplicateKeyException.DuplicateKeyException(Object, String)\n- uid: System.String\n  parent: System\n  isExternal: false\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)\n  parent: System.Data.Linq.DuplicateKeyException\n  isExternal: false\n  name: DuplicateKeyException(Object, String, Exception)\n  nameWithType: DuplicateKeyException.DuplicateKeyException(Object, String, Exception)\n  fullName: DuplicateKeyException.DuplicateKeyException(Object, String, Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: false\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Data.Linq.DuplicateKeyException.Object\n  parent: System.Data.Linq.DuplicateKeyException\n  isExternal: false\n  name: Object\n  nameWithType: DuplicateKeyException.Object\n  fullName: DuplicateKeyException.Object\n- uid: System.Data.Linq.DuplicateKeyException.#ctor*\n  parent: System.Data.Linq.DuplicateKeyException\n  isExternal: false\n  name: DuplicateKeyException\n  nameWithType: DuplicateKeyException.DuplicateKeyException\n  fullName: DuplicateKeyException.DuplicateKeyException\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n- uid: System.Data.Linq.DuplicateKeyException.Object*\n  parent: System.Data.Linq.DuplicateKeyException\n  isExternal: false\n  name: Object\n  nameWithType: DuplicateKeyException.Object\n  fullName: DuplicateKeyException.Object\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.Linq/DuplicateKeyException.xml\n- uid: System.Object.Equals(System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object)\n  nameWithType: Object.Equals(Object)\n  fullName: Object.Equals(Object)\n- uid: System.Object.Equals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object, Object)\n  nameWithType: Object.Equals(Object, Object)\n  fullName: Object.Equals(Object, Object)\n- uid: System.Object.GetHashCode\n  parent: System.Object\n  isExternal: false\n  name: GetHashCode()\n  nameWithType: Object.GetHashCode()\n  fullName: Object.GetHashCode()\n- uid: System.Exception.GetType\n  parent: System.Exception\n  isExternal: false\n  name: GetType()\n  nameWithType: Exception.GetType()\n  fullName: Exception.GetType()\n- uid: System.Object.MemberwiseClone\n  parent: System.Object\n  isExternal: false\n  name: MemberwiseClone()\n  nameWithType: Object.MemberwiseClone()\n  fullName: Object.MemberwiseClone()\n- uid: System.Object.ReferenceEquals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: ReferenceEquals(Object, Object)\n  nameWithType: Object.ReferenceEquals(Object, Object)\n  fullName: Object.ReferenceEquals(Object, Object)\n- uid: System.Exception.ToString\n  parent: System.Exception\n  isExternal: false\n  name: ToString()\n  nameWithType: Exception.ToString()\n  fullName: Exception.ToString()\n- uid: System.Exception.Data\n  parent: System.Exception\n  isExternal: false\n  name: Data\n  nameWithType: Exception.Data\n  fullName: Exception.Data\n- uid: System.Exception.GetBaseException\n  parent: System.Exception\n  isExternal: false\n  name: GetBaseException()\n  nameWithType: Exception.GetBaseException()\n  fullName: Exception.GetBaseException()\n- uid: System.Exception.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Exception\n  isExternal: false\n  name: GetObjectData(SerializationInfo, StreamingContext)\n  nameWithType: Exception.GetObjectData(SerializationInfo, StreamingContext)\n  fullName: Exception.GetObjectData(SerializationInfo, StreamingContext)\n- uid: System.Exception.HelpLink\n  parent: System.Exception\n  isExternal: false\n  name: HelpLink\n  nameWithType: Exception.HelpLink\n  fullName: Exception.HelpLink\n- uid: System.Exception.HResult\n  parent: System.Exception\n  isExternal: false\n  name: HResult\n  nameWithType: Exception.HResult\n  fullName: Exception.HResult\n- uid: System.Exception.InnerException\n  parent: System.Exception\n  isExternal: false\n  name: InnerException\n  nameWithType: Exception.InnerException\n  fullName: Exception.InnerException\n- uid: System.Exception.Message\n  parent: System.Exception\n  isExternal: false\n  name: Message\n  nameWithType: Exception.Message\n  fullName: Exception.Message\n- uid: System.Exception.SerializeObjectState\n  parent: System.Exception\n  isExternal: false\n  name: SerializeObjectState\n  nameWithType: Exception.SerializeObjectState\n  fullName: Exception.SerializeObjectState\n- uid: System.Exception.Source\n  parent: System.Exception\n  isExternal: false\n  name: Source\n  nameWithType: Exception.Source\n  fullName: Exception.Source\n- uid: System.Exception.StackTrace\n  parent: System.Exception\n  isExternal: false\n  name: StackTrace\n  nameWithType: Exception.StackTrace\n  fullName: Exception.StackTrace\n- uid: System.Exception.TargetSite\n  parent: System.Exception\n  isExternal: false\n  name: TargetSite\n  nameWithType: Exception.TargetSite\n  fullName: Exception.TargetSite\n"}